values.yaml

```diff
@@ -223,6 +223,27 @@ client:
     # https_proxy: http://localhost:3128,
     # no_proxy: internal.domain.com
 
+  # snaphotAgent contains settings for setting up and running snapshot agents
+  # within the Consul clusters. They are required to be co-located with Consul
+  # clients, so will inherit the clients' nodeSelector, tolerations and affinity.
+  # This is an Enterprise feature only.
+  snapshotAgent:
+    enabled: false
+
+    # replicas determines how many snapshot agent pods are created
```

Comment:
Going to leave this as-is for now since we don't seem to have solid guidelines for folks about how many of these to run at the moment. Since this is an enterprise feature, folks using it should have access to someone that will help guide them to a correct number.